config/config/sc-config.yaml

```diff
@@ -37,6 +37,7 @@ user:
   # Todo remove dependencie on alertmanager from service cluster
   alertmanager:
     enabled: false
+    namespace: monitoring
```

Comment:
So this is the namespace where the user-alertmanager runs?

Reply:
No, I guess that this is where the operator alertmanager runs in the service cluster. I think this was used by the wc alerts that we apply in the service cluster

Reply:
Okay, sure I did not think about that.